src/js/view/controls/controlbar.js

```diff
@@ -163,6 +163,13 @@ export default class Controlbar {
         }, localization.liveBroadcast);
         liveButton.element().textContent = localization.liveBroadcast;
 
+        const fullscreenButton = button('jw-icon-fullscreen', () => {
+            _api.setFullscreen();
+            const fullscreenText = _model.get('fullscreen') ? localization.exitFullscreen : localization.fullscreen;
+            fullscreenTip.setText(fullscreenText);
+            setAttribute(fullscreenButton.element(), 'aria-label', fullscreenText);
```

Comment:
seems weird that the resulting var (fullscreenButton) is also being accessed in the function. I would say use `elements.fullscreen.element()` instead